dependencies.yaml

```diff
@@ -42,7 +42,6 @@ dependencies:
           - &cmake_ver cmake>=3.23.1,!=3.25.0
           - c-compiler
           - cxx-compiler
-          - gdal>3.5.0,<3.6.0
```

Comment:
I would keep this line since gdal is required for building tests.

Reply:
We are trying to remove the gdal dependency for building libcuspatial, to ease creating wheels. So is this really necessary if it's only for building tests?

Reply:
This list should include everything needed to build libcuspatial from source. That includes tests, if we treat it the same as other RAPIDS packages.

This is used to generate a conda environment and does not affect wheel builds.

Reply:
Re-added this in 4d72fa2.